el/content_test.go

```diff
@@ -37,3 +37,15 @@ func TestP(t *testing.T) {
 		assert.Equal(t, `<p>hat</p>`, el.P(g.Text("hat")))
 	})
 }
+
+func TestBr(t *testing.T) {
+	t.Run("returns a br element in context", func(t *testing.T) {
+		assert.Equal(t, `<p>Test<br />Me</p>`, el.P(g.Text("Test"), el.Br(), g.Text("Me")))
+	})
+}
+
+func TestHr(t *testing.T) {
+	t.Run("returns a hr element with class", func(t *testing.T) {
+		assert.Equal(t, `<hr class="test" />`, el.Hr(g.Attr("class","test")))
```

Comment:
Oh, this hasn't been run through gofmt. Just add a space after the comma in `"class","test"`… 😅

Reply:
Sorry. I need to check how I make go fmt automatic when saving in GoLand. Should be fixed now.

Reply:
Yeah. I was editing from another project and this was not caught by my GoLand file watcher which runs `go fmt` on save. The details... Fixed now (and should not happen again).